src/model/project/assetFolders.ts

```diff
@@ -1,4 +1,16 @@
-/**
- * Generated by '@kontent-ai/model-generator@7.4.0'
- */
+
+/** 
+* This file has been auto-generated by '@kontent-ai/model-generator@8.0.0-19'.
+* 
+* (c) Kontent.ai
+*  
+* -------------------------------------------------------------------------------
+* 
+* Project: Kontent.ai Kickstart
+* Environment: Production
+* Id: 133f5ef8-66b1-0074-7d42-55371447aac4
```

Comment:
Can we have it without the ids? I don't think they are helpful.

Reply:
It helped me once to figure out, that the provided environment dindn't had good relations set 🗡️ 

Other thing is I tried but new model generator doesn't have the option :/ only if i generate it to the memory and remove it with regex